CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 project(SvalinnCommandPlugin)
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.7)
```

Comment:
is that bump in cmake requirement necessary ?

Reply:
This is required to use `VERSION_GREATER_EQUAL`

Reply:
This PR doesn't use `VERSION_GREATER_EQUAL` any more so I reverted this change